docs/build.md

```diff
@@ -0,0 +1,128 @@
+# Build instructions
+
+The pipeline requires Python version 3.5.
+
+## Python 3.5 installation (optional)
+The instructions below will be useful:
+* In case you have a newer version and need to run the pipeline you can follow the next steps to install Python 3.5
```

Comment:
I think this sounds more natural:
> * If you have a newer version of Python and want to install Python 3.5 without replacing your default `python` executable

Reply:
It does indeed! Replaced with your suggestion